front/src/routes/integration/all/netatmo/Dashboard-box-page/actions.js

```diff
@@ -0,0 +1,107 @@
+import update from 'immutability-helper';
```

Comment:
just one feedback on the folder name, we don't put uppercase character in folder name in Gladys, can you change that everywhere?